src/okta/okta.guard.ts

```diff
@@ -45,6 +51,20 @@ export class OktaAuthGuard implements CanActivate, CanActivateChild {
     });
   }
 
+  async canLoad(route: Route, segments: UrlSegment[]): Promise<boolean> {
+    this.onAuthRequired = route.data?.onAuthRequired || this.onAuthRequired;
+
+    const isAuthenticated = await this.oktaAuth.isAuthenticated();
+    if (isAuthenticated) {
+      return true;
+    }
+
+    const originalUri = segments[0].path;
```

Comment:
is `baseHref`(whenever it is set) part of `segments[0]` too?

Reply:
It's relative path, no baseHref is included.